setup.cfg

```diff
@@ -0,0 +1,5 @@
+[bdist_wheel]
+universal = 1
```

Comment:
This enables the build of universal wheels by default. Cf. https://packaging.python.org/tutorials/distributing-packages/#universal-wheels